src/thread.c

```diff
@@ -2217,6 +2217,53 @@ void ABTI_thread_print(ABTI_thread *p_thread, FILE *p_os, int indent)
     ABTU_free(prefix);
 }
 
+int ABTI_thread_print_stack(ABTI_thread *p_thread, FILE *p_os)
+{
+    void *p_stack = p_thread->attr.p_stack;
+    size_t i, j, stacksize = p_thread->attr.stacksize;
+    if (stacksize == 0 || p_stack == NULL) {
+        /* The thread is a main thread running on Pthreads, so we cannot
+         * dump the stack with p_stack. */
+        return ABT_ERR_THREAD;
+    }
+    const size_t value_width = sizeof(void *);
```

Comment:
I thought I already had asked you this but may be I forgot! Would be nice to also print the current stack pointer (p_thread->attr.p_stack) at this point to help analyze the stack content.

Reply:
Well forget this previous request to print p_thread->attr.p_stack, I better wanted to mean p_thread->ctx->fctx as the ULT current/saved stack pointer, if only fcontext mode is used/working as you have previously indicated to answer a previous request from me about this.

Reply:
Thanks! Both (`p_stack` and `fctx`) are printed in the latest commits.